compose.yml

```diff
@@ -9,10 +9,12 @@ suite:
               bundle exec rspec"
   environment:
     - DEBUG_LEVEL=0
-    - ST2_GITURL=${ST2_GITURL}
-    - ST2_GITREV=${ST2_GITREV}
-    - ST2PKG_VERSION=${ST2PKG_VERSION}
-    - ST2PKG_RELEASE=${ST2PKG_RELEASE}
+    ## For the following variables default values are used
+    #  unless explicitly defined:
+    # - ST2_GITURL=
```

Comment:
Why commented?
I need these vars to be passed from CI to Docker environment.

See: 
https://github.com/StackStorm/st2-packages/issues/12
https://github.com/StackStorm/st2-packages/pull/24#commitcomment-14646393